Commands/Monitor.php

```diff
@@ -35,6 +36,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $systemCheck->checkRedisIsInstalled();
         }
 
+        $iterations = $input->getOption('iterations');
+        if (empty($iterations) && $iterations !== 0 && $iterations !== '0') {
+            $queueId = null;
+        } elseif (!is_numeric($iterations)) {
+            throw new \Exception('iterations needs to be numeric');
+        } else {
+            $iterations = (int)$iterations;
+            if ($iterations <= 0) {
+              throw new \Exception('iterations needs to be a non-zero positive number');
+            }
+        }
+            $output->writeln("<info>Only running " . $iterations . " iterations.</info>");
```

Comment:
FYI: I think there are a few spaces too many, not too important though... just be good to maybe change if changing the PR anyway.